src/sentry/models/projectkey.py

```diff
@@ -72,6 +72,8 @@ class UseCase(enum.Enum):
     PROFILING = "profiling"
     """ An internal project key for submitting escalating issues metrics."""
     ESCALATING_ISSUES = "escalating_issues"
+    """ An internal project key for submitting events from tempest."""
+    TEMPEST = "tempest"
```

Comment:
you will need to create a django migration for this: https://develop.sentry.dev/backend/application-domains/database-migrations/

Django saves all this changes in migrations, although this will be no-op SQL. I suggest doing this in a separate PR, which will only contain this change and a migrations files created by django, as it will be easier to review it for owners-migrations team.